include/swift/AST/TypeCheckRequests.h

```diff
@@ -1627,6 +1627,25 @@ class SynthesizeDefaultInitRequest
   bool isCached() const { return true; }
 };
 
+/// Checks whether this declaration inherits its superclass' designated and
+/// convenience initializers.
+class InheritsSuperclassInitializersRequest
+    : public SimpleRequest<InheritsSuperclassInitializersRequest,
+                           bool(ClassDecl *), CacheKind::Cached> {
```

Comment:
We're gonna run this request on pretty much every class that gets referenced. I'm not sure we should be caching this in the evaluator quite yet.